exporter/jaeger/lib/opentelemetry/exporter/jaeger/collector_exporter.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'uri'
+
+module OpenTelemetry
+  module Exporter
+    module Jaeger
+      # An OpenTelemetry trace exporter that sends spans over HTTP as Thrift Binary encoded Jaeger spans.
+      class CollectorExporter
+        SUCCESS = OpenTelemetry::SDK::Trace::Export::SUCCESS
+        FAILURE = OpenTelemetry::SDK::Trace::Export::FAILURE
+        private_constant(:SUCCESS, :FAILURE)
+
+        def initialize(endpoint: ENV.fetch('OTEL_EXPORTER_JAEGER_ENDPOINT', 'http://localhost:14268'),
```

Comment:
[The spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-environment-variables.md#jaeger-exporter) is ambiguous here. It specifies a default of "http://localhost:14250" which is actually [the port for gRPC](https://www.jaegertracing.io/docs/1.19/deployment/#collectors). It isn't clear from the spec whether we should support Thrift Binary over HTTP (which is what we've implemented in this PR) or `model.proto` over gRPC. [The collector](https://github.com/open-telemetry/opentelemetry-collector/tree/master/receiver/jaegerreceiver#jaeger-receiver) supports both, along with Thrift Binary and Thrift Compact over UDP, [for compatibility with the Jaeger agent](https://www.jaegertracing.io/docs/1.19/deployment/#agent).